tests/test_blackjack_score.py

```diff
@@ -82,5 +82,22 @@ def test_returns_bust_for_scores_over_21():
   # Assert
   assert score == "Bust"
 
+def test_returns_12_for_ace_ace_king():
+    # Arrange
+  hand = ['Ace', 'Ace', 'King']
+
+  # Act
+  score = blackjack_score(hand)
 
+  # Assert
+  assert score == 12
 
+def test_returns_14_for_ace_ace_ace_ace():
+    # Arrange
+  hand = ['Ace', 'Ace', 'Ace', 'Ace']
+
+  # Act
+  score = blackjack_score(hand)
+
+  # Assert
+  assert score == 14
```

Comment:
[This PR](https://github.com/Ada-Developers-Academy/core-unit-1/pull/173/files) suggests adding these edge tests as optional for students who completed code to pass the tests above. Here's a suggestion for incorporating the test cases here:  

```suggestion
#*****Consider the following two edge test cases only after you have implemented code to pass all of the tests above*****
@pytest.mark.skip(reason="logic not yet implemented")
def test_returns_12_for_ace_ace_king():
    # Arrange
  hand = ['Ace', 'Ace', 'King']

  # Act
  score = blackjack_score(hand)

  # Assert
  assert score == 12

@pytest.mark.skip(reason="logic not yet implemented")
def test_returns_14_for_ace_ace_ace_ace():
    # Arrange
  hand = ['Ace', 'Ace', 'Ace', 'Ace']

  # Act
  score = blackjack_score(hand)

  # Assert
  assert score == 14
```

Reply:
If we like this, we should add it to the [tdd-exercise replit](https://replit.com/@adadev/tdd-exercise#main.py)

Reply:
LGTM!